pkg/tests/end_to_end_tests/ha_single_promscale_test.go

```diff
@@ -101,7 +105,9 @@ func prepareIngestorWithHa(db *pgxpool.Pool, t testing.TB) (*util.ManualTicker,
 	// so the calls to ha.Service.checkLeaseUntilAndLastWrite
 	// depends only on the max time seen by an instance
 	// and try_change_leader is called when we want
-	tooFarInTheFutureNowFn := func() time.Time { return time.Now().Add(time.Hour) }
+	tooFarInTheFutureNowFn := func() time.Time {
+		return time.Now().Add(time.Duration(atomic.AddInt32(&tCount, 1)) * time.Hour)
```

Comment:
why was this change needed?

Reply:
This is from a bug fix PR, will be removed from this commit once that PR is merged in (#569 )